src/test/java/uk/gov/companieshouse/model/validator/AppealReasonValidatorTest.java

```diff
@@ -10,10 +10,10 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 
-import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.model.Reason;
 
-@ExtendWith(SpringExtension.class)
+@ExtendWith(MockitoExtension.class)
 class AppealReasonValidatorTest {
 
     @InjectMocks
```

Comment:
No mocks to inject.

Reply:
refactored in new commit

Reply:
The **@InjectMocks** is still there. As Jonathan said, this needs to be removed as you're mocking the class you want to test so you're sort of not testing it when it should be a real instance. Either instantiate the private AppealReasonValidator object at declaration or probably better to have a setup method which instantiates before each test like this:

    @BeforeEach
    private void setup() {
    	appealReasonValidator = new AppealReasonValidator();
    }

Reply:
New commit resolving this pushed